toolboxes/hdg/coupledmixedpoisson_main.cpp

```diff
@@ -87,11 +87,13 @@ int main(int argc, char *argv[])
     int dimension = ioption(_name="case.dimension");
     std::string discretization = soption(_name="case.discretization");
 
-    auto dimt = hana::make_tuple(hana::int_c<2>,hana::int_c<3>);
+    auto dimt = hana::make_tuple(// hana::int_c<2>,
```

Comment:
why not 2D ?

Reply:
I don't know, it's @romainhild change in the last clean If I remember